dask_cuda/device_host_file.py

```diff
@@ -53,18 +84,15 @@ def __init__(
 
         self.host_func = dict()
         self.disk_func = Func(
-            partial(serialize_bytes, on_error="raise"), deserialize_bytes, File(path)
+            partial(serialize_bytelist, on_error="raise"), deserialize_bytes, File(path)
         )
         self.host_buffer = Buffer(
             self.host_func, self.disk_func, memory_limit, weight=weight
         )
 
         self.device_func = dict()
-        self.device_host_func = Func(
-            _serialize_if_device, _deserialize_if_device, self.host_buffer
-        )
```

Comment:
It looks like this disappeared.  I would expect `host_to_device` and `device_to_host` to be used here I think.

Reply:
Yes, my mistake when doing it while tired. I fixed that in the latest commits.